web/packages/teleport/src/services/integrations/integrations.ts

```diff
@@ -485,3 +486,18 @@ function makeAwsSubnets(json: any): Subnet {
     availabilityZone: availability_zone,
   };
 }
+
+export function getRdsEngineIdentifier(
```

Comment:
This functions converts `DatabaseEngine`, which comes from Discover, which is a specific part of the app, to `RdsEngineIdentifier`, which comes from `services/integrations` – a part of the app not bound to any specific feature.

As such, I think it'd be better to put this function into `web/packages/teleport/src/Discover/SelectResource/types.ts`. Otherwise we end up with a general service importing types from a very specific part of the app. If that part of the app wants to then use this service, we'll end up with a cyclic import.